### ISSUE TITLE
Import feature (probably) doesn't like long OR execute commands, basically it doesnt work with that:

### ISSUE THREAD

**Issue by LeafForge** [ezfe/tellraw#354]
i tried to import a command that i copied from my command block which looked like this:

execute as @a[scores={rules=1..}] at @s run tellraw @s ["",{"text":"========================================","bold":true,"strikethrough":true,"color":"gray"},"\n","\n",{"text":"          CookiedSite Rules:","color":"gold"},"\n","\n",{"text":"     1: No using bad words.","color":"yellow"},"\n",{"text":"     2: No advertising things.","color":"yellow"},"\n",{"text":"     3: No immature behavior.","color":"yellow"},"\n",{"text":"     4: No cheating or hacking.","color":"yellow"},"\n",{"text":"     5: Have common sense.","color":"yellow"},"\n",{"text":"     6: If something isn't a rule that doesn't mean","color":"yellow"},"\n",{"text":"     you are allowed to do that.","color":"yellow"},"\n","\n",{"text":"========================================","bold":true,"strikethrough":true,"color":"gray"}]

i needed to edit it for my server, but thats not important.
So basically i imported it and the website said something like: error occured, feel free to use the issue reporter.
so i did, and im here asking for help or fixing

Yes i did read the guidelines and my system is windows 10, and browser is normal google as far as im aware.

EDIT/UPDATE: i did try with shorter commands and even with commands with a slash before them but none seems to work, same error pops up i think theres something wrong with something :(

**Comment by ezfe** [comment]
Hey - sorry about this. The import only supports what you get when you hit the export button.

I have a work-in-progress to support what you’re doing but it’s not ready yet and I don’t have a timeline unfortunately. Sorry again

**Comment by ezfe** [comment]
#339
